src/locale/misc/smartSearch/en.yml

```diff
@@ -10,6 +10,7 @@ buttonLabels:
 filterTitles:
     all: All people
     most_active: A selection of the most active people
+    user: People who with a connection to a Zetkin user
```

Comment:
Phrasing is off here. I would suggest something along the lines of "People who used Zetkin" in this little title box.